core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -23,20 +23,23 @@
     </div>
   </div>
   <div class="oppia-content-wrapper">
-    <button class="btn oppia-audio-button oppia-audio-bar-button-transition"
+    <button class="btn oppia-audio-button oppia-audio-bar-button-transition protractor-accessibility-test-translation-start-record"
```

Comment:
Conventionally, we prefix class names used by protractor by protractor-test-. Why do you need to mention "accessibility" in the name?

Reply:
It was mention by @DubeySandeep in above reviews

Reply:
https://github.com/oppia/oppia/pull/6290/files/2c9f87305a59347a4298dcc64e647724a9b53b11#diff-04d7686be72cb72e89a095289c16790b

Reply:
I agree with @nithusha21 -- the convention followed is `protractor-test-`. @DubeySandeep, can you please explain the ideology behind this?  Thanks!

Reply:
@apb7, I didn't found any accessibility test on oppia and having `protractor-test-translation-start-record-accessibility` wasn't looking good to me so I asked to make it `oppia-accessibility-test-...`. I don't have any problem with `protractor-test-...` if you can suggest him a pattern to write an accessibility tests class name. Thanks!

Reply:
How about `protractor-test-accessibility-` or something on those grounds?

Reply:
Should I change it

Reply:
Yeah. Just make sure that the name is of the form with protractor-test-*, I would go by @apb7's suggestion to use protractor-test-accessibility-*.